cypress/support/e2e.js

```diff
@@ -18,3 +18,10 @@
 
 // Alternatively you can use CommonJS syntax:
 require('./commands')
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  console.log(err)
+  // returning false here prevents Cypress from
+  // failing a test when javascript in the browser fails
+  return false
```

Comment:
In the test where I'm proving that the javascript asset has not been loaded when the plugin is uninstalled, the test would fail before I could check the page had loaded without the javascript being run.  It's problematic.

Reply:
I can demo this for you if you wish.

Reply:
I've implemented your suggestion and it works.